alerter-component/src/main/java/io/github/steliospaps/ighackathon/instrumentpricealerter/alertercomponent/InitialTableScanner.java

```diff
@@ -70,7 +63,7 @@ void scanForTriggers() {
 								);
 
 					} else {
-						log.info("skipping");
+						log.info("ignoring (non trigger row)");
```

Comment:
cool. when we start processing INSTRUMENT entities, will we update this to that?

Reply:
Yeah we can refactor this part if we want to process instruments on initial startup